core/templates/domain/story/story-update.service.spec.ts

```diff
@@ -108,9 +108,9 @@ describe('Story update service', function() {
   it('should create a proper backend change dict for adding a prerequisite ' +
     'skill id to a node',
   function() {
```

Comment:
Change this and all other instances to anonymous functions (`() =>`)

Reply:
done!